lms/djangoapps/shoppingcart/processors/PayPal.py

```diff
@@ -107,7 +107,8 @@ def _record_purchase(params, order):
     # Mark the order as purchased and store the billing information
     payer = params.get('payer', {})
     payer_info = payer.get('payer_info', {})
-    billing_address = payer_info.get('billing_address', {})
+    # Data was formerly present under 'billing_address', but now it seems to be at 'shipping_address'
+    billing_address = player_info.get('billing_address') or payer_info.get('shipping_address', {})
```

Comment:
`player_info` should be `payer_info` i guess

Reply:
yes, my bad! fixed!